core/micro-services/workflow-compiling-service/src/main/scala/edu/uci/ics/texera/service/WorkflowCompilingService.scala

```diff
@@ -0,0 +1,40 @@
+package edu.uci.ics.texera.service
+
+import io.dropwizard.core.Application
+import io.dropwizard.core.setup.{Bootstrap, Environment}
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import edu.uci.ics.amber.util.PathUtils.workflowCompilingServicePath
+import edu.uci.ics.texera.service.resource.WorkflowCompilationResource
+
+class WorkflowCompilingService extends Application[WorkflowCompilingServiceConfiguration] {
+  override def initialize(bootstrap: Bootstrap[WorkflowCompilingServiceConfiguration]): Unit = {
+    // register scala module to dropwizard default object mapper
+    bootstrap.getObjectMapper.registerModule(DefaultScalaModule)
+  }
+
+  override def run(
+      configuration: WorkflowCompilingServiceConfiguration,
+      environment: Environment
+  ): Unit = {
+    // serve backend at /api/texera
+    environment.jersey.setUrlPattern("/api/texera/*")
```

Comment:
are you sure you want to map all `/api/texera/*`? I think it should be mapped to only compiling related endpoints.

Reply:
changed